docs/source/documentation/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+<!-- Do not delete this file it is the website entry point -->
+
+# Check if you can get legal aid technical documentation
+
+Check if you can get legal aid is a public facing service available on GOV.UK which gives members of the public clear guidance
+on whether they can receive civil legal aid for their problem, and provides guidance and next steps tailored to their outcome.
+
+The service provides a route for users to raise a case and optionally book an appointment with the Civil Legal Advice (CLA) call centre if their problem is in-scope for Civil Legal Advice
+and if the user financially qualifies for legal aid, however, this step may be skipped at this stage in certain cases.
```

Comment:
which step? I am assuming the financial eligibility check, if so can we be specific.